test/compflags/bradc/printstuff/version.goodstart

```diff
@@ -1 +1 @@
- version 1.27.0
\ No newline at end of file
+ version 1.28.0
```

Comment:
Oops, sophomore mistake:  The lack of linefeed here is important (at least, as the scripts for this test are currently written).  I'll fix this.